WORKING_GROUPS.md

```diff
@@ -0,0 +1,173 @@
+# IPFS Project Working Groups
+
+## Index
+
+- [Description](#description)
+- [Working Group Lifecycle](#working-group-lifecycle)
+- [Current Working Groups](#current-working-groups)
+- [Other](#other)
+
+## Description
+
+The IPFS Project Working Groups are teams of people that are appointed to research, develop and deploy new work under the scope that the working group is focused. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
+
+Each Working Group has its own cadence, lifecycle and objectives.
+
+## Working Group Lifecycle
+
+TODO: Define how a working group gets created, evaled and how to decide if its purpuse has been fulfilled and should be dismantled
+
+## Current Working Groups
+
+### Release
+
+The Release Working Group manages the releases for go-ipfs, js-ipfs and its internal dependencies.
+
+Responsibilities include:
+
+- Structure the released process.
+- Define the release Roadmap.
+- Decide what gets into each release.
+- Test releases.
+- Announce and gather feedback from the community on each release.
+
+### Specifications
+
+The Specifications Working Group coordinates and participates on the specification writting process.
```

Comment:
There will also be WGs for each spec or group of specs -- ie. who is on the IPLD Spec WG vs the libp2p spec WG.

Reply:
In the same way that the Internationalization Working Group participants are not the ones writing the translations to every language but rather create the platform and coordinate multiple contributors and possible vendors for those translations to exist, the Specifications working group should be more focused on ensuring that specs are written, clear, and that individuals that participate in other working groups and/or projects can effectively contribute to specs (through training, mentoring and so on). 

The Specifications working needs to have more of a role of an Editor + Reviewer, create a Peer Review system and structure the spec writing process (templates, coordination, timelines and objectives).